cog/downtime.py

```
@@ -0,0 +1,85 @@
+        json.dump([downtime.to_dict() for downtime in downtime_list], file, indent=4)
+
+
+async def get_history(
+    bot: Bot, channel_id, *, after: datetime, before: datetime | None = None
+):
+    if before is None:
+        before = datetime.now()
+    channel: Messageable = bot.get_channel(channel_id)
+
+    if not channel:
+        raise ValueError(f"Cannot get channel (id={channel}).")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)